src/components/node-list/node-list-row.js

```diff
@@ -163,6 +163,11 @@ const NodeListRow = memo(
           <input
             id={id}
             className="pipeline-nodelist__row__checkbox"
+            data-heap-event={
+              kind === 'element'
```

Comment:
QQ: why does kind === 'element' return focusMode event?

Reply:
To further @limdauto 's comment above, `kind === 'element'` will not indicate a focusMode event - The best way to return a focusMode event would be via the Visibility Icon, and in particular, the `kind` prop that dictates the type of visibility icon that gets rendered - this will effectively help us to differentiate between the eye icon ( for tracking if users clicked to see the individual visbility of nodes), and focusMode.